src/main/features/event-management/views/eventManagement.njk

```diff
@@ -214,7 +214,7 @@
           </div>
         </div>
         {% endif %}
-        {% if status == 'To Be Evaluated' %}
+        {% if status == 'to be evaluated' %}
```

Comment:
Please convert status to lower case because sometimes it will come in upper case then our condition will false.